libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -96,7 +96,8 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @Param{Plane, Rover}: REQUIRE
     // @DisplayName: Require Arming Motors 
     // @Description: Arming disabled until some requirements are met. If 0, there are no requirements (arm immediately).  If 1, sends the minimum throttle PWM value to the throttle channel when disarmed. If 2, send 0 PWM (no signal) to throttle channel when disarmed. On planes with ICE enabled and the throttle while disarmed option set in ICE_OPTIONS, the motor will always get THR_MIN when disarmed. Arming will occur using either rudder stick arming (if enabled) or GCS command when all mandatory and ARMING_CHECK items are satisfied. Note, when setting this parameter to 0, a reboot is required to immediately arm the plane.
-    // @Values: 0:Disabled,1:minimum PWM when disarmed,2:0 PWM when disarmed
+    // @Values{Plane}: 0:Disabled,1:minimum PWM when disarmed,2:0 PWM when disarmed
```

Comment:
It's an existing issue but just as a drive-by comment I'd say we could improve the wording of "1:minimum PWM when disarmed" because PWM doesn't clarify if its for RC input or servo output.  It should perhaps be changed to be "1:RC throttle at minimum" or something similar

Reply:
Definitely need tridge's input on that one.  It's the latter.

Reply:
yes, it's outside the scope of this PR, I'm just bringing it up as a drive-by comment